model-optimizer/extensions/middle/UpsampleToResample.py

```diff
@@ -124,17 +127,28 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
                                 get_height_dim(layout, input_shape_rank),
                                 get_width_dim(layout, input_shape_rank)])
 
-        resample_op = Interpolate(graph, dict(name=upsample_name + '/Interpolate',
-                                              axes=axes, mode=upsample.attrs()['mode'],
-                                              antialias=0, convert_to_resample=True)).create_node()
+        axes_node = Const(graph, {'name': upsample_name + '/axis_', 'value': axes}).create_node()
+
+        resample_op = Interpolate(graph, {'mode': upsample.attrs()['mode'], 'antialias': 0,
+                                          'convert_to_resample': True, 'pads_begin': int64_array([0]),
```

Comment:
The attribute `'convert_to_resample': True` is not needed anymore

Reply:
Fixed.